source/guides/deploy-dask-spark.md

```diff
@@ -0,0 +1,65 @@
+# How do Dask and Spark relate to deploying RAPIDS?
+
+Throughout the RAPIDS Deployment Documentation we spend a lot of time actually talking about deploying tools like [Dask](https://www.dask.org/) and [Apache Spark](https://spark.apache.org/). This document aims to answer common questions around _"What does deploying RAPIDS really mean?"_ and _"How do Dask and Spark relate to deploying RAPIDS?"_.
+
+To make things clearer let's start by answering questions like _"Which packages make up RAPIDS?"_ and _"Who is in the RAPIDS community?"_.
+
+## The many projects of RAPIDS
```

Comment:
This seems like a really useful way to convey to first-time users the world of RAPIDS (thinking of that PyData ecosystem image that gets passed around a lot), and I'm wondering if any of this can be applied to the [about section of RAPIDS' main page](https://rapids.ai/learn-more/#about), where some of these packages aren't mentioned?

Reply:
Thanks @charlesbluca. 

What do you think @exactlyallan? The goal of this doc is to help explain some backround about RAPIDS from a deployment perspective, but perhaps some of this copy would be useful to put on https://rapids.ai?